### ISSUE TITLE
Parameter Detail

### ISSUE THREAD

**Issue by paukes** [biogeochem/project_template#10]
A common metadata ask is to explain the parameter or column names (e.g. 'depth_m' = sample depth in meters below surface of water (m) ). 

Is this something we would want in the 'clean_data' folder? Or too much work?

**Comment by jjvenky** [comment]
Yes, this is an ongoing _issue_ that we need to address in a few ways.

In general, I think the metadata like that should be in one of the data folders since it's with the appropriate files.

Since we usually worth with a common set (20-30) of parameters, I do want to standardise the way we report. So one file with a) the common headers we use for variable names and b) the metadata descriptions is needed. I started a fe years ago and clearly have not finished it. We should take the header from a lake sampling chemistry file like we have from ELA and go from there. Together I think we could knock this out in half a day.

The longer-term process is going to be able methods for each of the paramters (analytes).
